### ISSUE TITLE
NordVPN Token Authentication for WireGuard

### ISSUE THREAD

**Issue by ProjectVRD** [qdm12/gluetun-wiki#26]
The method to extract a Private Key seems to be overly convoluted, can the image be updated to allow the use of the Token instead that NordVPN provides on the NordVPN user's account dashboard?

**Comment by qdm12** [comment]
I don't think this is possible for the time being because of how Gluetun is programmed to block all traffic not through the VPN, and that would require some exchange with the nordvpn API (AFAIK). We can keep this opened, this may change in the future (not weeks, more in a few months).
